### PR TITLE
[CARBONDATA-532] [WIP] fix bigdecimal compression issues when use_kettle=false

### PR DESCRIPTION
**Pull request by foryou2030**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[CARBONDATA-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).
 - [ ] Testing done
 
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - What manual testing you have done?
        - Any additional information to help reviewers in testing this change.
         
 - [ ] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA. 
                 
---

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/175/

**Comment by foryou2030** [comment]
ready to test

File: integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala [hunk 1]
```diff
@@ -436,17 +436,17 @@ case class LoadTable(
...
+//      val useKettle = options.get("useKettle") match {
+//        case Some(value) => value.toBoolean
+//        case _ =>
+//          val useKettleLocal = System.getProperty("use.kettle")
+//          if (useKettleLocal == null) {
+//            sqlContext.sparkContext.getConf.get("use_kettle_default", "true").toBoolean
+//          } else {
+//            useKettleLocal.toBoolean
+//          }
+//      }
```
Comment (ravipesala) [inline integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala]: why code is commented?

Comment (foryou2030) [inline integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala]: hi, @ravindra, the comment is for verify all test case when use_kettle=false.
and i found that, some commits introduced issues, due to lack of consideration of the new flow

**Comment by jackylk** [comment]
Please modify the description of this PR
And is this PR duplicated with #433 ?

**Comment by foryou2030** [comment]
@jackylk yes, duplicated , i will closed it.

**Pull request closed without merging**